nwbwidgets/analysis/placefields.py

```diff
@@ -55,7 +55,7 @@ def compute_speed(pos, pos_tt, smooth_param=40):
     return smooth(speed, smooth_param)
 
 
-def compute_2d_occupancy(pos, pos_tt, edges_x, edges_y, speed_thresh=0.03):
+def compute_2d_occupancy(pos, pos_tt, edges_x, edges_y, speed_thresh=0.03, velocity=[]):
```

Comment:
An optional variable usually takes default value `None` and that's what should be used here.